cmds/exp/pox/pox.go

```diff
@@ -247,8 +261,8 @@ func poxRun(args []string) error {
 		defer mountPoint.Unmount(0) //nolint:errcheck
 	}
 	for _, m := range chrootMounts {
-		mp, err := mount.Mount(m.source, filepath.Join(dir, m.target),
-			m.fstype, m.data, m.flags)
+		v("mount(%q, %q, %q, %v, %1, %v", m.source, filepath.Join(dir, m.target), m.fstype, m.data, m.flags)
```

Comment:
Were some of these debugging prints or do we want to leave them in?

Reply:
Those have proven very handy for seeing what's going on, so I'd like to keep them. They're easier than strace.